usagov/usagov-2021
# USAGOV-1340-Homepage-cards-accessible-names

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1340

## Description

Remove aria-label from links in All Topics cards and Life Events Carousel cards the so the accessible name will be determind based on the card content.


## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions


* Visit the homepage
* Inspect the All Topics cards and Life Events carousel cards
* Check the <a> element of the card
* Confirm that there is no aria-label attribute
* Optional:
  * Check the accessible name of the <a> tags using the accessibility panel.
  * Confirm that the accessible name includes the card's title and description content

### Requires New Config

### Requires New Content

### Validation Steps
- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
